connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -120,7 +155,8 @@ public SalesforceController(
             @Value("${sf.opportunityContactLinkPath}") String sfOpportunityContactLinkPath,
             @Value("${sf.opportunityTaskLinkPath}") String sfOpportunityTaskLinkPath,
             @Value("${sf.attachmentTasklinkPath}") String sfAttachmentTasklinkPath,
-            @Value("${sf.opportunityFieldsUpdatePath}") final String sfOpportunityFieldsUpdatePath
+            @Value("${sf.opportunityFieldsUpdatePath}") final String sfOpportunityFieldsUpdatePath,
+            @Value("${vmware.se.cards.flag}") final String vmwSECardsFlag
```

Comment:
Flag can be loaded directly on a Boolean.
Also I think it'll be better to provide a default as `False`.

Reply:
Done.